<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1251</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion client</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>190</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>EMAIL</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>52</x>
        <y>310</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>DATE_CREATION</string>
      </property>
     </widget>
     <widget class="QPushButton" name="updateButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>260</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>NOM</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>310</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sortByNameButton">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>20</y>
        <width>101</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier par Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>240</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>445</x>
        <y>50</y>
        <width>561</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>260</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sortButton">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>260</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Actualiser</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>310</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEditDateCreation">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>310</y>
        <width>110</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sortByIdButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier par ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showClientCountButton">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>260</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Statistiques</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CLIENT_ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>240</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TELEPHONE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="insertButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>350</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>260</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Taches</string>
     </attribute>
     <widget class="QLineEdit" name="lineEditChatBot">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>181</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditChatHistory">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>1231</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="chatBotSendButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>ChatBot</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sendSmsButton">
      <property name="geometry">
       <rect>
        <x>1110</x>
        <y>20</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Envoyer message</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>20</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>MSG Ã  envoyer</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>20</y>
        <width>331</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sendButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>AVG TEMP</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>187</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>id client</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_13"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1259</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
